apps/ejabberd/c_src/ejabberd_zlib_drv.c

```diff
@@ -42,6 +42,40 @@ typedef struct {
       z_stream *i_stream;
 } ejabberd_zlib_data;
 
+/* Wrappers around driver_alloc() that check  */
+/* for OOM.                                   */
+void erl_exit(int n, char* v, ...) { abort(); }
```

Comment:
I'm not sure if `abort` is the best choice here. How about [`driver_failure`](http://erlang.org/doc/man/erl_driver.html#driver_failure)?